docs/modules/record-encryption/con-record-encryption-overview.adoc

```diff
@@ -0,0 +1,95 @@
+// file included in the following:
+//
+// assembly-record-encryption-filter.adoc
+
+[id='con-topic-encryption-overview-{context}']
+= Encryption process
```

Comment:
Maybe we could align it with the other heading `How Kroxylicious works` like`How Record Encryption works`.